mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(value) {
+  const result = value * 1.2;
```

Comment:
Hi, the name of the variable "result" is good but it can be improved by using a more realistic name line finalPrice / salesPrice. I appreciate your calculation technique. Keep it up